scss/titles.scss

```diff
@@ -0,0 +1,18 @@
+.title-container {  
+    height: 90px;
+    width: 100px;
+
+    h2 {
+        left: 40px;
+        position: relative;
+        text-align: center;
+        top: 20px;   
+    }
+
+    &::before{
+        background: url('./assets/triangle.svg') 50px;
+        content: '';
+        @include image-size(50px);
```

Comment:
Good! this is excellent! in the future you will see that this can be used to auto generate margins, paddings..etc